pages-css/cart-css/page-layout.css

```diff
@@ -0,0 +1,123 @@
+@import url("/pages-css/compo-css/navbar.css");
+@import url("/pages-css/compo-css/card.css");
+@import url("/pages-css/cart-css/page-style.css");
+
+/* color variables */
+
+:root {
+    --color-footer-background: #cecece;
+    --color-red: #fc0101;
+    --color-white: #ffffff;
+    --color-dark: #050505;
+}
+
+
+/* *** */
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Orienta', sans-serif;
+    font-family: 'Quicksand', sans-serif;
+    scroll-behavior: smooth;
+}
+
+
+/* page container */
+
+.page-layout-container {
+    display: grid;
+    gap: 0.313rem;
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 0.313rem;
+    row-gap: 0.313rem;
+    grid-template-areas: "header header header " "section-1 section-1 section-1 " "section-2 section-2 section-3" "footer footer footer ";
+    padding: 0.313rem;
+    padding-left: 0.938rem;
+    padding-right: 0.938rem;
+}
+
+
+/* *** */
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    grid-area: header;
+    height: 3.75rem;
+}
+
+.section-1 {
+    grid-area: section-1;
+    padding-left: 1.563rem;
```

Comment:
i dont think you have to write upto 3 places of decimals. keep it limited to 1 max 2.

Reply:
ok